compiler/damlc/base-hoogle-template.txt

```diff
@@ -9,4 +9,4 @@
 @package daml-stdlib
 @version 1.2.0
```

Comment:
random thought: Perhaps these version numbers should track the SDK? (Separate issue/PR of course)

Reply:
Yeah I noticed that as well. I’m not quite sure what this version number actually is used for.

Reply:
FWIW `1.2.0` does not seem to appear anywhere on the current Hoogle results. No idea what these are used for.